app.js

```diff
@@ -113,14 +113,48 @@ closeIcon2.addEventListener('click', () => {
   mainContainer1.classList.remove('show');
 });
 
-const contactFormCont = document.getElementById('formcontact');
+// FORM VALIDATION
 contactFormCont.addEventListener('submit', (event) => {
   const emailInput = contactFormCont.email.value;
   if (emailInput.toLowerCase() !== emailInput) {
     event.preventDefault();
     const errorTag = contactFormCont.getElementsByTagName('small');
-    errorTag[0].innerHTML = 'Please insert email address in lowercase!';
+    errorTag[0].innerHTML = 'Please insert email in lowercase!';
   } else {
     contactFormCont.action = 'https://formspree.io/f/mqkngenv';
   }
-});
\ No newline at end of file
+});
+
+// Local Storage
```

Comment:
- [ ] [OPTIONAL] It would be nice to add another javascript file for this milestone, this will make debugging and also future upgrades easier.